File: src/main/java/guru/xgm/language/angelscript/formatting/AngelScriptCodeStyleSettings.java
package guru.xgm.language.angelscript.formatting;

import com.intellij.psi.codeStyle.CodeStyleSettings;
import com.intellij.psi.codeStyle.CustomCodeStyleSettings;
import lombok.experimental.FieldNameConstants;
import org.jetbrains.annotations.NotNull;

// https://projectlombok.org/features/experimental/FieldNameConstants
@FieldNameConstants
public class AngelScriptCodeStyleSettings extends CustomCodeStyleSettings {
    public AngelScriptCodeStyleSettings(@NotNull CodeStyleSettings container) {
        super("AngelScriptCodeStyleSettings", container);
    }

    public boolean AT_ENUM_EQ = false;
}

File: src/main/kotlin/guru/xgm/language/angelscript/formatting/AngelScriptCodeStyleSettings.kt
@file:Suppress("PropertyName")

package guru.xgm.language.angelscript.formatting

import com.intellij.psi.codeStyle.CodeStyleSettings
import com.intellij.psi.codeStyle.CustomCodeStyleSettings
import lombok.experimental.FieldNameConstants

class AngelScriptCodeStyleSettings(container: CodeStyleSettings) :
    CustomCodeStyleSettings("AngelScriptCodeStyleSettings", container) {
    @JvmField
    var AT_ENUM_EQ: Boolean = false
}

